### DISCUSSION TITLE
1970s/1970/pdp7unix/

### DISCUSSION THREAD

**Discussion by giscus** [InstallerLegacy/screenshots.virtualhub.eu.org#17] [Comments]
# 1970s/1970/pdp7unix/

! PDP7 Unix was an operating system for the DEC PDP-7 computer. It is the earliest available Unix version to be able run in an emulator.

https://screenshots.virtualhub.eu.org/1970s/1970/pdp7unix/

**Comment by pulkitkrishna00** [comment]
`dmr` stands for "Dennis MacAlistair Ritchie", while `ken` stands for "Kenneth Lane Thompson". They were the original developers of Unix. Dennis Ritchie was also the main developer of the C programming language (He is the R in K&R C).
